### ISSUE TITLE
`ip netns show` do not print a name space "qrouter".

### ISSUE THREAD

**Issue by TsutomuNakamura** [tsuna-server/build-server-ansible#45]
Command `ip netns show` do not print a name space `qrouter` but only `qdhcp` after network components were created.
```
# ip netns show
qdhcp-xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx (id: 0)
```
It is supposed to also print a `qrouter`.

https://github.com/tsuna-server/build-server-ansible/blob/312c7332a64234e32b7cfbccefd5101fd2fa1674/roles/test_env/tasks/test_created_network/main.yml#L1-L8

**Comment by TsutomuNakamura** [comment]
This problem was caused due to failed to start `neutron-l3-agent`.

```
# systemctl status neutron-l3-agent
● neutron-l3-agent.service - OpenStack Neutron L3 agent
     Loaded: loaded (/lib/systemd/system/neutron-l3-agent.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Sun 2021-12-05 17:13:34 UTC; 8min ago
       Docs: man:neutron-l3-agent(1)
    Process: 3875 ExecStart=/etc/init.d/neutron-l3-agent systemd-start (code=exited, status=1/FAILURE)
   Main PID: 3875 (code=exited, status=1/FAILURE)

Dec 05 17:13:34 dev-private-router01 systemd[1]: neutron-l3-agent.service: Main process exited, code=exited, status=1/FAILURE
Dec 05 17:13:34 dev-private-router01 systemd[1]: neutron-l3-agent.service: Failed with result 'exit-code'.
Dec 05 17:13:34 dev-private-router01 systemd[1]: neutron-l3-agent.service: Scheduled restart job, restart counter is at 5.
Dec 05 17:13:34 dev-private-router01 systemd[1]: Stopped OpenStack Neutron L3 agent.
Dec 05 17:13:34 dev-private-router01 systemd[1]: neutron-l3-agent.service: Start request repeated too quickly.
Dec 05 17:13:34 dev-private-router01 systemd[1]: neutron-l3-agent.service: Failed with result 'exit-code'.
Dec 05 17:13:34 dev-private-router01 systemd[1]: Failed to start OpenStack Neutron L3 agent.

```

**Comment by TsutomuNakamura** [comment]
About FWaaS.
https://docs.openstack.org/neutron/queens/admin/fwaas-v2-scenario.html

Related article.
https://www.ntt-tx.co.jp/column/tec/180702/

**Comment by TsutomuNakamura** [comment]
Is a package `neutron-fwaas-common` needed on Ubuntu?
```
# apt-get install neutron-fwaas-common
```
The installation will create a config file `/etc/neutron/fwaas_driver.ini` and it makes `neutron-l3-agent` can be run normally.

**Comment by TsutomuNakamura** [comment]
Fixed it at 5243c95.
